### ISSUE TITLE
Reduce repository size

### ISSUE THREAD

**Issue by tpluscode** [dotnetrdf/dotnetrdf#5]
I think we could investigate why the repository is so big after fresh checkout. It also takes quite some time to clone.

I've seen the same issue with the previous Mercurial repo and it seems that there are some large files which contribute to that

**Comment by kal** [comment]
Yeah, I did some digging into that. Very early on in the repo history there are some big Turtle files added. They did get gzipped later, but I *suspect* that the repo is still carrying the original Turtle files around. It is possible to edit the repo history to delete those files, but I think I would want to be sure that it really does help with the repo size.

Of course, if going forwards we decide to develop 2.0 from a separate dotnetrdf2 repo then we could probably avoid this issue completely by starting development from a snapshot of the current 1.9 branch.
